src/routing_membrane.rs

```diff
@@ -434,6 +435,34 @@ impl<F> RoutingMembrane<F> where F: Interface {
         };
     }
 
+    /// If routing table is not empty, the find group request is sent through available node(s),
+    /// else if bootstrap_endpoint is non-empty, request sent through bootstrap_endpoint.
+    /// If routing table is empty & bootstrap_endpoint is empty, attempt is made to
+    /// send the request through the endpoint acheived from re-bootstrapping.
+    fn send_find_group_request(&mut self) {
+        let our_name = self.own_name.clone();
+        let find_group_msg = self.construct_find_group_msg(&our_name);
+        let serialised_msg = match encode(&find_group_msg) {
+            Ok(msg) => msg,
+            Err(_) => panic!("Failure to serialise find group message.")
```

Comment:
just as below, have the function return a result that we can ignore( ) for now, but throw an error, don't panic